docs/index.js

```diff
@@ -1,7 +1,7 @@
 //---------------Smooth Scrolling---------------//
 $(function() {
 	$('a[href*="#"]:not([href="#"])').click(function() {
```

Comment:
We might be able to exclude the two FAQ URLS with the following:
`$('a[href*="#"]:not([href="#"]):not([href="#parents"]):not([href="#retailers"])').click(function() {...`